csrankings-y.csv

```diff
@@ -621,7 +619,9 @@ Yuan Feng 0001,University of Technology Sydney,https://www.uts.edu.au/staff/yuan
 Yuan He 0004,Tsinghua University,http://www.greenorbs.org/people/heyuan,5ojW7_UAAAAJ
 Yuan Hong,University of Connecticut,https://yhongcs.github.io,KJuZW2wAAAAJ
 Yuan Jiang 0001,Nanjing University,http://lamda.nju.edu.cn/jiangy,Qp_v9woAAAAJ
+Yuan Luo 0002,Middlesex University,https://www.mdx.ac.uk/about-us/our-people/staff-directory/profile/luo-yuan-roger,NOSCHOLARPAGE
 Yuan Luo 0003,Shanghai Jiao Tong University,http://www.cs.sjtu.edu.cn/en/PeopleDetail.aspx?id=155,NOSCHOLARPAGE
+Yuan Roger Luo,Middlesex University,https://www.mdx.ac.uk/about-us/our-people/staff-directory/profile/luo-yuan-roger,NOSCHOLARPAGE

```

Comment:
This entry is not a current alias per DBLP of Yuan Luo 0002; please ask them to merge it.